File: koncol/laba13.2(pr2,lb2)/pr3.c
#include <math.h>
#include <stdio.h>

int r(int q){
    return q*3.14/180;
}

int main()
{
    for(int x=0;x<=90;x+=10){
        printf("sin %d = %lf |",x,sin(r(x)));
        printf("cos %d = %lf\n",x,cos(r(x)));
        
    }
    return 0;
}

File: console/laba3(pr3,lb3)/pr3.cs
using System;

class HelloWorld
{
  static void Main ()
  {
    for(int x=10;x<90;x+=10)
    Console.WriteLine(x+" gr "+"sin="+ Math.Round(Math.Cos(x),2) + " cos=" + Math.Round(Math.Sin(x),2));
  }
}
